docs/UserGuide.md

```diff
@@ -184,6 +184,18 @@ Examples:
 * `list -c` followed by `delete -c 2,3` deletes the 2nd and 3rd contact in Athena.
 * `find -c n/Betsy` followed by `delete -c 1` deletes the 1st contact in the results of the `find` command.
 
+#### Deleting contacts by Tag : `deletetag`
```

Comment:
This was not changed along with the command word.

Reply:
resolved